src/AqBanking/Command/ShellCommandExecutor.php

```diff
@@ -18,7 +20,7 @@ public function execute($shellCommand)
         $returnVar = null;
         $tempFile = tempnam(sys_get_temp_dir(), 'aqb-');
 
-        exec($shellCommand . ' 2>' . $tempFile, $output, $returnVar);
+        $this->pid = exec($shellCommand . ' 2>' . $tempFile ." & echo $!;", $output, $returnVar);
```

Comment:
Okay, now this is interesting. .... What are you trying to accomplish with this?
Is this actually working? The shellCommand will run in the background and will we still get the error output?

Reply:
Looks like this was removed in https://github.com/Mestrona/aqbanking-php/pull/11/commits/1842efdbc000c47b1627cbbfc74423685f34889e, right?

Reply:
Correct - this was a test if I could do some blocking operation, when I get the process id. But then I thought its not needed since - when I don't send the operation to the background it is actually already blocking.